src/server/scripts/Commands/cs_learn.cpp

```diff
@@ -318,33 +320,30 @@ class learn_commandscript : public CommandScript
 
     static bool HandleLearnAllLangCommand(ChatHandler* handler, char const* /*args*/)
     {
+        Player* player = handler->getSelectedPlayerOrSelf();
         // skipping UNIVERSAL language (0)
         for (uint8 i = 1; i < LANGUAGES_COUNT; ++i)
-            handler->GetSession()->GetPlayer()->LearnSpell(lang_description[i].spell_id, false);
+            player->LearnSpell(lang_description[i].spell_id, false);
 
         handler->SendSysMessage(LANG_COMMAND_LEARN_ALL_LANG);
         return true;
     }
 
-    static bool HandleLearnAllDefaultCommand(ChatHandler* handler, char const* args)
+    static bool HandleLearnAllDefaultCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* target;
-        if (!handler->extractPlayerTarget((char*)args, &target))
```

Comment:
was this allowing to specify the player target as command parameter ? does the new code change this behavior ?